src/main/java/frc/robot/subsystems/Vision.java

```diff
@@ -0,0 +1,86 @@
+package frc.robot.subsystems;
+
+import org.photonvision.PhotonCamera;
+import org.photonvision.targeting.PhotonPipelineResult;
+import org.photonvision.targeting.PhotonTrackedTarget;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class Vision extends SubsystemBase
+{
+    private static Vision  _instance;
+    
+    public static Vision getInstance()
+    {
+        if (_instance == null) 
+        {
+            _instance = new Vision();
+        }
+       
+        return _instance;
+    }
+   
+    private final PhotonCamera _camera0 = new PhotonCamera("camera1");
+    private final PhotonCamera _camera1 = new PhotonCamera("camera2");
+    private PhotonPipelineResult _latestResult0 = new PhotonPipelineResult();
+    private PhotonPipelineResult _latestResult1 = new PhotonPipelineResult();
+    
+   
+    private Vision()
+    {
+
+    } 
+
+    public boolean hasTargets0()
+    {
+        return _latestResult0.hasTargets();
+    }
+    
+    public boolean hasTargets1()
+    {   
+        return _latestResult1.hasTargets();
+    }
```

Comment:
I don't like the naming convention we're using for the two cameras, and by extension, these methods. I would at the absolute least expect these to be called `camera0HasTargets` and `camera1HasTargets`